packages/valory/connections/abci/tests/test_fuzz/base.py

```diff
@@ -61,11 +65,13 @@ class BaseFuzzyTests(AEATestCaseMany):
     agent_process = None
     cli_log_options = ["-v", "INFO"]
 
-    AGENT_TIMEOUT = 10
+    AGENT_TIMEOUT_SECONDS = 10
 
     @classmethod
-    def setUpClass(cls) -> None:
+    def setup_class(cls) -> None:
         """Sets up the environment for the tests."""
+
+        super().setup_class()
```

Comment:
test had both a `setUpClass` and a `setup_class` method. I think `setup_class` was executed first, but decided to unify them.